src/components/web/PinkButton.jsx

```
@@ -1,6 +1,6 @@
 import PropTypes from 'prop-types'
 
-const PinkButton = ({ label, ...props }) => {
+const PinkButton = ({ label, type: button, ...props }) => {
```

ESLint

[error] 'button' is defined but never used.

(no-unused-vars)